challenge-1/bmi-comparator.js

```diff
@@ -0,0 +1,18 @@
+class Person {
+
+    constructor(mass, height) {
+        this._mass = mass;
+        this._height = height;
+    }
+
+    get bodyMassIndex() {
+        return this._mass / (this._height * this._height)
+    }
+}
+
+const mark = new Person(78, 1.69);
+const john = new Person(92, 1.95);
```

Comment:
Can you give me some inside on `const`, `let` and `var`? 😅 
No need to change just asking for a friend 😄

Reply:
`const` and `let` declare variables with a scope. The scope can be of level of block, function or global. `const` declare variables that only can be assigned one time and `let` variables that can be assigned several times. `var` declare always global variables, those variable can be access from any spot. A variable declared in a function can be  accessed from out the function and a variable declare in a if can be accessed from out the if, etc.